src/core/viz/parser.js

```diff
@@ -179,3 +179,14 @@ function cleanJsep() {
     jsep.addLiteral('true');
     jsep.addLiteral('false');
 }
+
+/**
+ * Remove comments from string
+ * - // one-line comments
+ * - /* multi-line
+ *     comments */
+export function cleanComments(str) {
+    return str
+        .replace(/\/\/.*/g, '')
+        .replace(/\/\*(\*(?!\/)|[^*])*\*\//g, '');
+}
```

Comment:
Mmmm, what about "//" and "/*" inside quotes?

Reply:
I see. We need stronger regex :muscle: